packages/sign_in_with_apple/example/lib/main.dart

```diff
@@ -1,6 +1,7 @@
 import 'dart:io';
 
 import 'package:flutter/material.dart';
+// ignore: import_of_legacy_library_into_null_safe
```

Comment:
TODO: Revisit before merge.

(I assume `http` will be updated by then, and then this becomes obsolete)